src/alcaramba/Drawable/Town/Town.tscn

```diff
@@ -43,8 +48,30 @@ margin_right = 185.0
 margin_bottom = 231.0
 script = ExtResource( 2 )
 
+[node name="ContainerRight" type="VBoxContainer" parent="."]
+material = SubResource( 3 )
+margin_left = 695.0
+margin_top = 314.0
+margin_right = 1050.0
+margin_bottom = 912.0
+
+[node name="CurrentTile" type="VBoxContainer" parent="ContainerRight"]
+margin_right = 355.0
+margin_bottom = 50.0
+rect_min_size = Vector2( 0, 50 )
+
+[node name="SpareTiles" type="VBoxContainer" parent="ContainerRight"]
```

Comment:
Needs to be size-capped to the end of the screen and needs a scrollbar, so you can access "off-screen" tiles

Reply:
Just for prototyping. UX and design has to done in future task